src/content/manager/data-pool.manager.ts

```diff
@@ -1,12 +1,15 @@
 import { BaseManager } from "./base.manager";
 import { ManagerConfig } from "../../interfaces/manager-config.interface";
+import {logger} from "../../util/logger";
+import {DataPoolSlimTransport} from "../../interfaces/data-pool-manager.interfaces";
 
 export class DataPoolManager extends BaseManager {
     public static DATA_POOL_FILE_NAME_PREFIX = "data-pool_";
     private static API_URL = "/integration/api";
     private static DATA_POOL_PUSH_URL = DataPoolManager.API_URL + "/pool-import";
     private static DATA_POOL_ACTIONS_URL = DataPoolManager.API_URL + "/pools/{id}";
     private static DATA_POOL_PULL_URL = DataPoolManager.DATA_POOL_ACTIONS_URL + "/export";
+    private static DATA_POOLS_LIST_URL = DataPoolManager.API_URL + "/pools/paged";
```

Comment:
I think these changes are not needed now.

Reply:
Reverted these changes 👍